process/block/sovereignChainBlock.go

```diff
@@ -840,11 +850,19 @@ func (scbp *sovereignChainBlockProcessor) processSovereignBlockTransactions(
 		return nil, err
 	}
 
+	// TODO: Marius C.: Implement signing this message
+	_ = scbp.collectOutGoingOperations()
```

Comment:
no need for this message to be added in the block?

Reply:
It is going to be added in next PR here: https://github.com/multiversx/mx-chain-go/pull/5673/files#diff-6af228a15da3fe0b40476a5a20ade8f283e2be09bc144907650a055a8824c69fR863